model/src/main/kotlin/PackageCuration.kt

```diff
@@ -74,6 +91,8 @@ data class PackageCuration(
             it.showStackTrace()
         }.getOrDefault(false)
 
+    private fun String.isVersionRange() = versionRangeIndicators.any { contains(it, ignoreCase = true) }
```

Comment:
The code in second in (line 77 onwards) internally does create a range list like this:
```
val curationVersion = "[2.0.0, 2.0.2]"
val rangesList = RangesListFactory.create(curationVersion)
println(rangesList.get().size)
```
This code prints `0` as output due to the white space after `,` in `[2.0.0, 2.0.2]` .
I believe it would be nice to somehow act if the range list size is `0` while there is 
range list chars in the string. If both conditions are met, we could:

1. make  `isVersionRange()` return false
2. probably to hard: throw an exception

What do you think?

Reply:
I added a check for the case when `id.version` looks like a version range, but semver4j creates an invalid version range with the size of `0`.